examples/test_api.c

```diff
@@ -555,10 +557,24 @@ double getbw(char* name, char* buf, int times)
        * Timing how long this takes can be used to estimate
        * interference from a background flush. */
       if (reduce > 0) {
-//        work_cpu(timestep, reduce);
-        work_allreduce_iters(timestep, reduce);
-//        work_allreduce(timestep, reduce);
-//        work_memcpy(timestep, reduce, 1024L * 1024L * 1024L);
+        switch (kernel) {
+          case 0: {
+            work_cpu(timestep, reduce);
+            break;
+          }
+          case 1: {
+            work_memcpy(timestep, reduce, 1024L * 1024L * 1024L);
+            break;
+          }
+          case 2: {
+            work_allreduce(timestep, reduce);
+            break;
+          }
+          case 3: {
+            work_allreduce_iters(timestep, reduce);
+            break;
+          }
+        }
```

Comment:
Please add a `default` case that will print an error if `kernel` is invalid

Reply:
Added.